.github/workflows/lint_markdown.yml

```diff
@@ -15,7 +15,7 @@ jobs:
       - name: Vale
         uses: errata-ai/vale-action@v2
         with:
-          vale_flags: "--glob=!{plugins/source/.snapshots/*,CHANGELOG.md,.github/styles/proselint/README.md}"
+          vale_flags: "--glob=!{plugins/source/testdata/*,CHANGELOG.md,.github/styles/proselint/README.md}"
```

Comment:
You might need to ignore it a in the `lint-structure` job too. It probably ignored it by default being a `.` file